tests/test_mujoco.py

```diff
@@ -0,0 +1,47 @@
+from stable_baselines.common.policies import MlpPolicy
+from stable_baselines.common.vec_env import DummyVecEnv
+from stable_baselines import PPO2
+
+import gym
+import benchmark_environments
+
+def test_fixed_env_model_as_good_as_gym_env_model():
```

Comment:
Let's discuss in person about the best way of testing this. I like having this kind of test suite we can run, it may be too expensive to run as part of unit tests, but we can just add an `expensive` marker to skip it by default.